plugin/evm/vm.go

```diff
@@ -308,110 +324,115 @@ func (vm *VM) Initialize(
 		log.Warn("Chain enabled `AllowFeeRecipients`, but chain config has not specified any coinbase address. Defaulting to the blackhole address.")
 	}
 
-	// Handle offline pruning
```

Comment:
Just to confirm, this was just a duplicate block?

Reply:
yep